src/bindings/python/_flux/_core_build.py

```diff
@@ -0,0 +1,38 @@
+from cffi import FFI
+import re
+
+ffi = FFI()
+
+
+ffi.set_source(
+    "_flux._core",
+    """
+#include <src/include/flux/core.h>
+
+
+void * unpack_long(ptrdiff_t num){
+  return (void*)num;
+}
+// TODO: remove this when we can use cffi 1.10
```

Comment:
So close! TOSS3 has cffi 1.9.1 installed for python3.6 :sob:

Reply:
I saw that... so, so close...